posts/2021-12-15-liberty-guide-log4j-cve.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Liberty Guide on Log4j2 CVE-2021-44228"
```

Comment:
I know this is only applicable to log4j2, but I think we should drop the '2' from the title and description as I think most places reference it as the `log4j` vulnerability (likewise the jars are like `log4j-core` don't have a 2).  Also, most of the blog references it as `Log4J`
```suggestion
title: "Liberty Guide on Log4j CVE-2021-44228 (Log4Shell)"
```